cli/src/main/resources/help/user.adoc

```diff
@@ -70,6 +74,12 @@ Grants one or more roles to a principal.
 *user roles deny* --roles='role1'[,'role2'...] 'principal'::
 Denies one or more roles to a principal.
 
+*user roles create* --permissions='perm1'[,'perm2'...] 'role'::
+Creates a new role with the supplied permissions.
+
+*user roles remove* 'role'::
+Removes an existing role.
```

Comment:
Hi @tristantarrant, when I try to remove a role I get "Not Found: Resource not found". I checked this with Don and he's facing the same issue. Can you please take a look?

Reply:
```
[t480s-43036@cluster//containers/default]> user roles ls
["observer","cleric","application","admin","monitor","wizard","deployer"]

[t480s-43036@cluster//containers/default]> user roles remove wizard
Not Found: Resource not found
```

Reply:
Fixed